cache/blobs_linux.go

```diff
@@ -71,10 +72,11 @@ func (sr *immutableRef) tryComputeOverlayBlob(ctx context.Context, lower, upper
 		}
 	}()
 
+	bufW := bufio.NewWriterSize(cw, 131072)
```

Comment:
Any suggestions for the buffer size? Tried 64KiB but the it made the export slower in +20-30% range.
https://github.com/ktock/buildkit/actions/runs/1301780687

Reply:
nit: write it as `128 * 1024`

Reply:
Thank you for the review. Fixed.